charts/devtron/templates/NOTES.txt

```diff
@@ -1,45 +1,48 @@
 
-1. Run the following command to get the default admin password. Default username is admin
+1. Run the following command to get the password for the default `admin` user:
 
-   kubectl -n devtroncd get secret devtron-secret -o jsonpath='{.data.ACD_PASSWORD}' | base64 -d
-{{- if $.Values.components.devtron.ingress.enabled }}
+   kubectl -n devtroncd get secret devtron-secret \
+   -o jsonpath='{.data.ACD_PASSWORD}' | base64 -d {{- if $.Values.components.devtron.ingress.enabled }}
```

Comment:
`{{- if $.Values.components.devtron.ingress.enabled }}` seems to be repeated here